app/Http/Controllers/PresetsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class PresetsController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return string
+     */
+    public function index()
+    {
+        // All these events take 1 day and repeat yearly
+        $yearly = json_decode('[
```

Comment:
Why use json format as we convert them immediately to a PHP array?

Reply:
To be fair, they were previously defined in JavaScript and copied from there

Reply:
They will be refactored anyway?

Ok for now then.

(Using an array gives us the opportunity to translate the labels…).

Reply:
Yes it will, this will either use an open data source (hopefully where the translations are already present) or it'll use a library (if we can't find an open data source) to calculate the holidays (so we don't have to manually add them each year)

Reply:
* https://holidayapi.com/
* https://packagist.org/packages/hansott/holidays